cime/config/e3sm/machines/config_compilers.xml

```diff
@@ -86,29 +86,32 @@ flags should be captured within MPAS CMake files.
 
 <compiler COMPILER="cray">
   <CFLAGS>
-    <append compile_threaded="FALSE"> -h noomp </append>
+    <append compile_threaded="TRUE"> -h omp </append>
+    <append DEBUG="TRUE"> -g -O0 </append>
   </CFLAGS>
   <CPPDEFS>
     <!--http://docs.cray.com/cgi-bin/craydoc.cgi?mode=View;id=S-3901-83;idx=books_search;this_sort=;q=;type=books;title=Cray%20Fortran%20Reference%20Manual -->
     <append> -DFORTRANUNDERSCORE -DNO_R16 -DCPRCRAY</append>
-    <append COMP_NAME="moby"> -DDIR=NOOP </append>
   </CPPDEFS>
   <FC_AUTO_R8>
     <base> -s real64 </base>
   </FC_AUTO_R8>
   <FFLAGS>
-    <base> -O2  -f free -N 255  -h byteswapio -em </base>
-    <append compile_threaded="FALSE"> -h noomp </append>
-    <append DEBUG="TRUE"> -g -trapuv  -Wuninitialized </append>
+    <base> -f free -N 255 -h byteswapio -em </base>
+    <append compile_threaded="TRUE"> -h omp </append>
+    <append compile_threaded="FALSE"> -M1077 </append>
+    <append DEBUG="TRUE"> -g -O0 </append>
   </FFLAGS>
   <FFLAGS_NOOPT>
     <base> -O0 </base>
   </FFLAGS_NOOPT>
   <HAS_F2008_CONTIGUOUS>TRUE</HAS_F2008_CONTIGUOUS>
   <LDFLAGS>
     <base> -Wl,--allow-multiple-definition -h byteswapio </base>
-    <append compile_threaded="FALSE"> -h noomp </append>
+    <append compile_threaded="TRUE"> -h omp </append>
```

Comment:
Just trying to understand: What's the motivation for the switch for noomp to omp? The old version didn't have any threaded builds?

Reply:
`noomp` is compiler's default, turning on `omp` only for threaded builds.